src/main/kotlin/com/r3/logicapps/BusMessageHandler.kt

```diff
@@ -32,9 +32,9 @@ class BusMessageHandler(
             handleRequest(workbenchAdapter.transformIngress(payload), message.lockToken)
         } catch (exception: IngressFormatException) {
```

Comment:
Can we have an additional catch with a wider scope (Exception, Throwable) and to get an idea of what exactly is causing this? Can't tell from the logs, which is weird.

Reply:
I can log the type in the finally block if that helps?

Reply:
Sure, any additional info will help. Stacktrace + message